Conversion error:  error: failed to find symbol 'nullptr'

I just tried Kani for the first time, and followed the [installation guide](https://model-checking.github.io/kani/install-guide.html).

While testing the installation with `kani test.rs`, I get this error:

```console
Kani Rust Verifier 0.61.0 (standalone)
warning: target feature `x87` must be enabled to ensure that the ABI of the current target can be implemented correctly
  |
  = note: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
  = note: for more information, see issue #116344 <https://github.com/rust-lang/rust/issues/116344>

warning: target feature `sse2` must be enabled to ensure that the ABI of the current target can be implemented correctly
  |
  = note: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
  = note: for more information, see issue #116344 <https://github.com/rust-lang/rust/issues/116344>

warning: 2 warnings emitted

/usr/lib/gcc/x86_64-pc-linux-gnu/15.1.1/include/stddef.h:465:1: error: failed to find symbol 'nullptr'
   typedef __typeof__(nullptr) nullptr_t;
CONVERSION ERROR
error: goto-cc exited with status exit status: 1
```

Rust version: 1.85.1-x86_64-unknown-linux-gnu
GCC version: 15.1.1
OS: Arch Linux

--- a/library/kani/kani_lib.c
+++ b/library/kani/kani_lib.c
@@ -10,8 +10,6 @@ void *memcpy(void *dst, const void *src, size_t n);
 void *calloc(size_t nmemb, size_t size);
 void *malloc(size_t size);
 
-typedef __CPROVER_bool bool;
-
 /// Mapping unit to `void` works for functions with no return type but not for
 /// variables with type unit. We treat both uniformly by declaring an empty
 /// struct type: `struct Unit {}` and a global variable `struct Unit VoidUnit`
@@ -22,13 +20,13 @@ extern struct Unit VoidUnit;
 // `assert` then `assume`
 #define __KANI_assert(cond, msg)            \
     do {                                    \
-        bool __KANI_temp = (cond);          \
+        __CPROVER_bool __KANI_temp = (cond);          \
         __CPROVER_assert(__KANI_temp, msg); \
         __CPROVER_assume(__KANI_temp);      \
     } while (0)
 
 // Check that the input is either a power of 2, or 0. Algorithm from Hackers Delight.
-bool __KANI_is_nonzero_power_of_two(size_t i) { return (i != 0) && (i & (i - 1)) == 0; }
+__CPROVER_bool __KANI_is_nonzero_power_of_two(size_t i) { return (i != 0) && (i & (i - 1)) == 0; }
 
 // This is a C implementation of the __rust_alloc function.
 // https://stdrs.dev/nightly/x86_64-unknown-linux-gnu/alloc/alloc/fn.__rust_alloc.html

